### ISSUE TITLE
Checkboxes - indent multiline labels

### ISSUE THREAD

**Issue by adam-scania** [scania/corporate-ui#422]
![checkbox-multiline](https://user-images.githubusercontent.com/18661799/49640089-eeef1e80-fa0c-11e8-8720-fab789795587.PNG)

In this case the area marked in yellow should be a whitespace margin, indenting the text in a straight line.

**Comment by adam-scania** [comment]
```
<label class="checkbox-inline">
    <input type="checkbox">
    <span></span>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
</label>
```

**Comment by scania-bootstrap** [comment]
Even though we haven't seen such a use case (quite a long label for a tight, unresponsive, space). Although, I suspected this issue to show up. My suggestion is that you'll have a look if it can be solved with a "hanging indent": https://www.thesitewizard.com/css/hanging-indents.shtml

**Comment by adam-scania** [comment]
That solution will offset the checkbox if applied on the `<label>`, or will require a block element if applied to the text only (which will create an empty line in the beginning next to the checkbox).

I can make it almost work like this (there's a space character next to the checkbox still):
```
<script src="https://d31jnweo1ynb8u.cloudfront.net/build/global/3.4.0-CF/js/corporate-ui.js?preload=false"></script>
<style>
.checkbox-inline {
    width: 150px;
    padding-left: 35px;
}

.checkbox-inline > input[type="checkbox"] + span {
}

.checkbox-inline > input[type="checkbox"] + span::before {
    margin-left: -30px;
}
</style>
<label class="checkbox-inline">
    <input type="checkbox">
    <span></span>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
</label>
```

Anyhow, UX wants this to be the default behavior everywhere.

**Comment by scania-bootstrap** [comment]
Both checkboxes and radios should since 3.4.0 be surrounded with a div. Hope that's in the releasenotes, I will check. Try this (it's not pixel adjusted):

.form-group .checkbox label {
    padding-left: 20px;
    text-indent: -9px;
    line-height: 1.4;
}

              <div class="checkbox">              
                <label class="checkbox-inline" style="width: 300px">
                    <input type="checkbox">
                    <span></span>
                    Agree License type of text. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
                </label>  
               </div>

**Comment by yuyo2** [comment]
#457

**Comment by yuyo2** [comment]
Fixed in #475
